---
layout: page
title: Research
subtitle:  
---

Broadly speaking, I am interested in the study of evolution, of how and why the amazing variety of lifeforms we see all around us came to be the way they are. Mutation, recombination, genetic drift, gene flow and selection are all fundamentally important factors which can influence evolution. Understanding the interplay of these various processes is my main research interest.

More specifically, I am interested in the process of selection and how we can leverage DNA sequence data to make inferences about selective pressures faced by organisms. 

A lot of the research I have carried out has been on the house mouse (*Mus musculus*). For research in evolutionary genetics, mice are an excellent model system. There are very well developed genetic/genomic resources available for mice and, in addition, they have naturally very large population sizes which, for a variety of reasons, makes them suitable for evolutionary analysis.

I am currently a post-doc at the University of British Columbia working with Mike Whitlock and Sam Yeaman at the University of Calgary. I am working as part of the CoAdapTree project (http://coadaptree.forestry.ubc.ca/). One of the aims of CoAdapTree is to identify genes involved in local adaptation in several coniferous tree species. Under climate change, tree species may face novel  environments, ones for which they are not adapted. As such, identifying the genes which underly locally adapted phenotypes may help guide the development of programs of assisted gene flow. 

My role in the CoAdapTree project is to help develop and test methods which can be used to detect local adaptation and to quantify the extent to which evolutionarily distinct lineages have used the routes to adapt to variable climates. 

If you spend enough time with me, I'm bound to start talking about selective sweeps and background selection. I am very interested in how these two processes affect variation in genetic diversity in eukaryotes, and specifically, what their footprints can tell us about moleular evolution.


## Current Projects

### Global adaptation in structured populations

I am currently working on a project looking at how global adaptation affects out ability to detect selecton using genome scans. In particular, we are asking whether recurrent positive selection generates *Fst* outliers in genome scans aimed at identifying local adaptation. So far we have found that, yes, global adaptation can indeed influence genomes scans. The plot below shows an *Fst* Manhattan plot for a parapatric population pair.


![](/img/other/combined_3panel.jpg "Global adaptation in structured populations")

Selective sweeps of globally beneficial alleles can cause epehemeral peaks of *Fst* (see Slatkin and Wiehe 1998 *Genet. Res.*; Kim and Maruki 2011 *Genetics*; Bierne 2012 *Evolution*; Feder *et al* 2019 - *currently BioRXiv*). The bottom left panel on the above figure shows the change in allele frequency over time for several beneficial mutations (the solid and dashed lines indicate the two populations of the parapatric pair). The bottom-right panel shows *Fst* over time for an analysis window which contains the genes in which the selected alleles occurred. The plot shows the traces for a handful of beneficial mutations, I have removed most to keep the plot tidy, but even with this sample, it is clear that there is a reasonable chance of observing an ephemeral *Fst* peak at any point in time. 
 
The extent to which it does depends on the distribution of fitness effects for new beneficial mutations and that is pretty much unknown for natural populations.

This project is part of my postdoc with Mike Whitlock at UBC and Sam Yeaman at the University of Calgary.

### Adaptive molecular evolution in murid rodents

![](/img/research/exons_divFam_genetic_castaneus.png "Trough Mania!")

This project carries on where my 2018 MBE paper left off. In that paper, one of the main conclusions was that background selection on its own could not generate the troughs in diversity seen in *Mus musculus castaneus*, but that selective sweeps as generated by the parameters of the DFE estimated from the site frequency spectrum couldn't either. The plot above shows that the relative reductions in nucleotide diversity around exons are of a similar magnitude in other sub-species of *Mus musculus* as well as the sister species *Mus spretus*.

A way forward is to use the patterns of diversity to estimate parameters of positive selection that are consistent with observed data.

This work is in collaboration with researchers at the Universities of Edinburgh and Cambridge.

